src/lang/pt-BR.json

```diff
@@ -46,6 +46,7 @@
   "app_dotacoach": "Pergunte a um treinador",
   "app_pvgna": "Encontre um guia",
   "app_pvgna_alt": "Encontre um guia de Dota 2 na Pvgna",
+  "app_rivalry": "Bet on Pro Matches",
```

Comment:
This is in English, this should be: "Aposte em partidas Pro"

Reply:
you can submit updates on https://translate.opendota.com